packages/actions-global/src/hooks/use-on-action-error.spec.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { Action } from 'redux';
+import { Provider } from 'react-redux';
+import { renderHook } from '@testing-library/react-hooks';
+import useOnActionError from './use-on-action-error';
+
+jest.mock('../actions/handle-action-error', () => (payload: Error) => ({
+  type: 'fake-handle-action-error',
+  payload,
+}));
+
+const renderUseOnActionError = () => {
+  const dispatchedActions: Array<Action> = [];
+  const fakeStore = {
+    dispatch: (action: Action) => dispatchedActions.push(action),
+    getState: () => null,
+    subscribe: () => null,
+  };
+  const { result } = renderHook(() => useOnActionError(), {
+    // eslint-disable-next-line react/display-name
+    wrapper: ({ children }) => (
+      // @ts-ignore: partially implemented Redux Store is OK as a mock
+      <Provider store={fakeStore}>{children}</Provider>
+    ),
+  });
+  return { onActionError: result.current, dispatchedActions };
+};
```

Comment:
Interesting and concise way of testing it. Would have probably tested the whole app to see if something appears on screen. Or use the good old `redux-mock-store`.

Anyway, all cool! :)